Oryx.Cognite/src/Playground/Handler.fs

```diff
@@ -0,0 +1,147 @@
+// Copyright 2019 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+
+/// Common types for the SDK.
+namespace Oryx.Cognite.Playground
+
+open System
+open System.Collections.Generic
+open System.Net.Http
+open System.Text.Json
+open System.Threading
+open System.Threading.Tasks
+
+open FSharp.Control.Tasks.V2.ContextInsensitive
+
+open Oryx.Cognite
+
+open Oryx
+open Oryx.Retry
+open Oryx.SystemTextJson
+open Oryx.SystemTextJson.ResponseReader
+open Oryx.Protobuf
+open Oryx.Protobuf.ResponseReader
+
+open CogniteSdk
+
+
+// Shadow types for Oryx that pins the error type to ResponseError
```

Comment:
We should not need to duplicate these shadow types. Can we use from top-level Handler.fs or should we put them in Types.fs?